lib/base-controller.js

```diff
@@ -1,9 +1,24 @@
 'use strict';
 
-var Controller = require('hmpo-form-wizard').Controller;
+var Controller = require('hmpo-date-controller');
 var util = require('util');
 var _ = require('lodash');
 
+// In HOF we are more lenient with date inputs than in vanilla
+// hmpo-form-wizard and so must override some validators
```

Comment:
Happy to remove if others agree. My rationale was that it's a strange thing to do and so needed some explanation.